addons/im_livechat/static/src/core/web/discuss_sidebar_category_patch.js

```diff
@@ -44,7 +44,7 @@ const DiscussSidebarChannelPatch = {
     get itemNameAttClass() {
         return {
             ...super.itemNameAttClass,
-            "fst-italic text-muted fw-normal": this.thread.livechat_end_dt,
+            "fst-italic text-muted fw-normal overflow-visible": this.thread.livechat_end_dt,
```

Comment:
Is that meant to cancel an overflow-hidden?

Reply:
Yes, I'm changing the fix to not interrupt with `text-truncate`.